telecolor_convert.c

```diff
@@ -0,0 +1,52 @@
+/*
+ ***************************************************************************************************
+ * Author:             Harshitha Subramani
+ * Date:               23/02/2020  
+ * Filename:           main.c
+ *****************************************************************************************************
+*
+***************************************************************************************
+* Includes
+***************************************************************************************
+*/
+#include <stdio.h>
+#include <assert.h>
+#include "colorpair.h"
+
+/*This Function prints the translated Colorpair names from Pairnumber
+ * \param     colorpair             Pointer to Major and Minor color pair
+ * \param     buffer                buffer storing color strings                     */
+
+
+void ColorPairToString(const ColorPair* colorPair, char* buffer) {
```

Comment:
Check consistency - should this function start with Get... like the others in this file?

Reply:
Corrected.